astropy/units/tests/test_quantity_typing.py

```diff
@@ -0,0 +1,107 @@
+# coding: utf-8
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Test the Quantity class and related."""
+
+import sys
+import typing as T
+
+import pytest
+
+from astropy import units as u
+
+""" The Quantity class will represent a number + unit + uncertainty """
+
+
+class TestQuantityTyping:
+    """Test Quantity Typing Annotations.
+
+    .. todo::
+
+        - switch tests to ``get_type_hints``, ``get_args``, ``get_origin``
```

Comment:
Left-over comments?

Reply:
In python 3.8+ these become useful functions. Just leaving it as a note for next version bump.